apps/asap-cli/src/import/protocols/insert.ts

```diff
@@ -99,6 +99,7 @@ export default async (data: Protocol): Promise<void> => {
     sharingStatus: { iv: 'Network Only' as ResearchOutputSharingStatus },
     asapFunded: { iv: 'Not Sure' as DecisionOption },
     usedInAPublication: { iv: 'Not Sure' as DecisionOption },
+    pmsEmails: { iv: null },
```

Comment:
this may make tsc happy but it doesn't look valid. We do not actually insert pm emails, do we, so i think that the type of the object is incorrect. Perhaps we need to omit this property in order to not have to provide it?

Reply:
Fixed with the new types